### PR TITLE
Create scheme

### PR DESCRIPTION
**Pull request by aliveGUY**
# Created Scheme 
* Created clinic database 
* Created patients tabel
* Created medical_history tabel
* Created invoices tabel
* Created invoice_items tabel
* Created treatment tabel
* Created treatment_history tabel

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    CONSTRAINT tratments_id PRIMARY KEY(id)
+);
+ALTER TABLE invoice_items ADD CONSTRAINT tratments_id FOREIGN KEY(treatment_id) REFERENCES tratments(id);
+
+CREATE TABLE treatment_history (
+    medical_history_id INT,
+    treatment_id INT,
+    CONSTRAINT medical_histories FOREIGN KEY(medical_history_id) REFERENCES medical_histories(id),
+    CONSTRAINT tratments_id FOREIGN KEY(treatment_id) REFERENCES tratments(id)
+);
```
Comment (DeliceLydia) [inline schema_based_on_diagram.sql:54]: - [ ] Please create `indexes` on all `FOREIGN KEYS`
example: `CREATE INDEX ON medical_histories (patient_id);`...etc

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights:
- Awesome README
- Followed git workflow


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+CREATE INDEX ON medical_histories (patient_id);
+CREATE INDEX ON invoices (medical_history_id);
+CREATE INDEX ON invoice_items (invoice_id);
+CREATE INDEX ON invoice_items (treatment_id);
+CREATE INDEX ON treatment_history (medical_history_id);
+CREATE INDEX ON treatment_history (medical_history_id);
```
Comment (TedLivist) [inline schema_based_on_diagram.sql:61]: - [ ] I think one of these should be:

  ```
  CREATE INDEX ON treatment_history (treatment_id);
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @aliveGUY and @alexpuente0,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi team,

## APPROVED 🟢

Great implementation of all the requirements in this milestone. 👏

✔️ Creating indexes for foreign keys

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**